qiita_db/data.py

```diff
@@ -615,6 +616,54 @@ def remove_filepath(self, fp):
         # Delete the files, if they are not used anywhere
         purge_filepaths(conn_handler)
 
+    def status(self, study):
+        """The status of the raw data within the given study
+
+        Parameters
+        ----------
+        study : Study
+            The study that is looking to the raw data status
+
+        Returns
+        -------
+        str
+            The status of the raw data
+
+        Raises
```

Comment:
You might want to remove this section as there are not specific exceptions raised in this method.